sql/branch.sql

```diff
@@ -0,0 +1,4 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE amount = 0 AND due != 0;
+    UPDATE payment_instructions SET due = floor(9.81/(amount)) * amount WHERE due > 9.81;
```

Comment:
Help me out with the 9.81 ... I don't recognize it as the minimum amount minus credit card fees, or something like that. Where does it come from?